### DISCUSSION TITLE
Dependent config for cyclc time

### DISCUSSION THREAD

**Discussion by HSLife1991** [apache/dolphinscheduler#16411] [Q&A]
Hi,
    Recently i am using the feature 'dependent' of dolphinscheduler, and i config two task, such as taskA, taskB, both are hourly tasks.
    taskB is depend on taskA, the cyclic time for taskB is set for hourly, last3Hour。 But when taskA is running successful, taskB is blocked。
    
    
![ds dependent](https://github.com/user-attachments/assets/ae9d9050-a1d7-4140-a2d4-c198ff9966c9)

**Comment by HSLife1991** [comment]
may i ask what is the meaning of cyclic time and rule for hourly?

**Reply by Tandoy** [reply]
Cyclic time set to the past three hours means that it will only check for success if there have been successful task dependencies within the past three hours.
